dbt/models/spatial/spatial.stadium.sql

```diff
@@ -2,9 +2,30 @@
     config(materialized='table') 
 }}
 
-SELECT
-    stadium.name,
-    ST_ASBINARY(ST_POINT(stadium.lon, stadium.lat)) AS geometry,
-    ST_ASBINARY(ST_POINT(stadium.x_3435, stadium.y_3435)) AS geometry_3435,
-    CAST(stadium.year AS VARCHAR) AS year
-FROM {{ ref('spatial.stadium_raw') }} AS stadium
+WITH distinct_years AS (
+    SELECT DISTINCT CAST(year AS INTEGER) AS year
+    FROM {{ source('spatial', 'parcel') }}
+
+),
+
+stadium_years AS (
+    SELECT
+        stadium.name,
+        stadium.lon,
+        stadium.lat,
+        stadium.x_3435,
+        stadium.y_3435,
+        years.year
+    FROM {{ ref('spatial.stadium_raw') }} AS stadium
+    CROSS JOIN distinct_years AS years
+    WHERE CAST(stadium.year AS INTEGER) <= years.year
+)
+
+SELECT DISTINCT
+    stadium_years.name,
+    CAST(stadium_years.year AS VARCHAR) AS year,
+    ST_ASBINARY(ST_POINT(stadium_years.lon, stadium_years.lat)) AS geometry,
+    ST_ASBINARY(ST_POINT(stadium_years.x_3435, stadium_years.y_3435))
+        AS geometry_3435
+FROM stadium_years
+ORDER BY year, name;
```

Comment:
**nitpick:** Try not to include semi-colons in your queries, as sometimes they break the composability of queries in dbt.

```suggestion
ORDER BY year, name
```